### PR TITLE
Add extra layers to model and full model scoring at end

### PR DESCRIPTION
**Pull request by matthew-sochor**
A few enhancements from this afternoon

### CODE REVIEW

File: modeling/train_head.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 
-from keras.layers import Dense, Dropout, Input, BatchNormalization
+from keras.layers import Dense, Dropout, Input, BatchNormalization, Dense
```
Comment (thenomemac) [inline modeling/train_head.py]: Dense is listed twice here, use a text editor with a linter :D

Comment (matthew-sochor) [inline modeling/train_head.py]: linter shminter, I'll fix and merge

**Comment by thenomemac** [comment]
curious what was the final ACC you got? Did you figure out the argmax based accuracy disconnect issue?
